lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.html

```diff
@@ -0,0 +1,32 @@
+<adf-search-filter-tabbed
+    [settings]="settings"
```

Comment:
To be honest I was not sure which approach is better. I see advantages for both approaches. For your approach @MichalKinas these which you mentioned in your comment, on the other hand I see advantage with this approach that when we move some common code to SearchFilterTabbed then we won't need repeat that code for every jira (dates, users etc.). 

But I guess we will lose that advantage after addressing your comment here: https://github.com/Alfresco/alfresco-ng2-components/pull/8714/files/2b64a8d3f5e153dfdcd5988a338ab5d095f64f6a#r1245270433 

because you suggested there that we don't need that whole logic for translations so in that case there won't be sense anymore to pass these extra data to SearchFilterTabbed because there won't be too much logic. 

So fine for me, let's change it as you suggested after addressing mentioned comment too.

Reply:
Updated the code. The Tabbed component now no longer has any properties/event emitters. All of the logic of computing queries/display labels/fields are now moved to the SearchDateRangeAdvancedTabbedComponent